iron-scroll-target-behavior.html

```diff
@@ -189,15 +189,21 @@
      * Scrolls the content to a particular place.
      *
      * @method scroll
-     * @param {number} left The left position
-     * @param {number} top The top position
+     * @param {number|!ScrollToOptions} leftOrOptions The left position or scroll options
```

Comment:
FYI (especially @aomarks) i dont think theres a way (yet, if ever) to specify overloads. so technically `scroll(obj, 1)` is valid according to this. not a lot we can do about that as jsdoc/closure types don't seem to have any such way

Reply:
Nope, pretty sure there's no way to describe that to Closure.